core/story-api/src/main/java/edu/cmu/cs/dennisc/animation/affine/AffineAnimation.java

```diff
@@ -66,7 +66,6 @@ public AbstractTransformable getSubject() {
   }
 
   public void setSubject(AbstractTransformable sgSubject) {
-    assert sgSubject != null;
```

Comment:
If the value is null it triggers a different red queen that should be avoided as well.
More elaboration on https://phab.alice.org/T1090